templates/setup/docker-compose.override.yml

```diff
@@ -0,0 +1,143 @@
+version: "3.9"
+
+services:
+
+  # # common settings for all our services:
+  # common_config:
+  #   deploy:
+  #     replicas: 1
+  #     update_config:
+  #       parallelism: 1
+  #       order: start-first
+  #       failure_action: rollback
+  #       delay: 10s
+  #     rollback_config:
+  #       parallelism: 0
+  #       order: stop-first
+  #     restart_policy:
+  #       condition: any
+  #       delay: 5s
+  #       max_attempts: 3
+  #       window: 120s
+
+  #nginx setup
+  web:
+    ports:
+      - "<%=port%>:80"
+      - "443:443"
+    # extends:
+    #   service: common_config
+
+
+  #db: use Maria DB as our backend DB
+  <%= dbExpose ? "" : "# " %>db:
+  <%= dbExpose ? "" : "# " %>  ports:
+  <%= dbExpose ? "" : "# " %>    - "<%=dbPort%>:3306"
+  <%= dbExpose ? "" : "# " %>  # extends:
+  <%= dbExpose ? "" : "# " %>  #   service: common_config
```

Comment:
Can the port numbers be read from the .env?

Reply:
Yes, both port numbers are available in the `.env` file:
```
# External port numbers
WEB_PORT=<%= port %>
DB_PORT=<%= dbPort %>
```

We could simply update this to reference the ENV variable.

Do you know what the proper reference here would be for the ENV variable?